content/rs/concepts/high-availability/clustering.md

```diff
@@ -171,4 +171,4 @@ the following limitations:
     but do not result in the proper violation error being returned.
 - **Renaming keys**: The use of the RENAME / RENAMENX commands is
     allowed only when the key's original and new values are mapped to
-    the same tag.
```

Comment:
@Paz-Yanover @AlonMagrafta Are these changes from "tags" to "slots" correct?

Reply:
I find it right to change to slots from tags